packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardBuilder.ts

```diff
@@ -89,7 +89,7 @@ export default class AdaptiveCardBuilder {
         textblock[prop] = template[prop];
       }
 
-      textblock.speak = text;
+      this.card.speak += `${text} `;
```

Comment:
Could you explain a bit about this?

I am only scanning thru the code, I thought concatenating a string should look something like ``speak += ` ${text}`;``, with a space prepended to it, instead of having the space appended to it, like ``speak += `${text} `;``.

Reply:
I was amusing the speak attribute would be an empty string and since it's only being modified there it made more sense to me to append the space to the end. I think either would be fine. Do you have a preference?